src/components/activity-page/managed-interactive/managed-interactive.test.tsx

```diff
@@ -210,4 +212,87 @@ describe("ManagedInteractive component", () => {
+      is_hidden: false,
+      is_half_width: true,
+      show_in_featured_question_report: true,
+      inherit_aspect_ratio_method: true,
+      custom_aspect_ratio_method: "DEFAULT",
+      inherit_native_width: true,
+      custom_native_width: 576,
+      inherit_native_height: true,
+      custom_native_height: 435,
+      inherit_click_to_play: true,
+      custom_click_to_play: false,
+      inherit_full_window: true,
+      custom_full_window: false,
+      inherit_click_to_play_prompt: true,
+      custom_click_to_play_prompt: null,
+      inherit_image_url: true,
+      custom_image_url: null,
+      library_interactive: {
+        hash: "af5d1860b2d4a037ae01e3d86931a30886fcb42d",
+        data: {
+          aspect_ratio_method: "DEFAULT",
+          authoring_guidance: "",
+          base_url: "https://models-resources.concord.org/question-interactives/branch/master/multiple-choice/",
+          click_to_play: false,
+          click_to_play_prompt: null,
+          description: "A basic multiple choice interactive. This is pointing to the master branch and is in development so it shouldn't be used by real activities. \r\n",
+          enable_learner_state: true,
+          full_window: false,
+          has_report_url: false,
+          image_url: null,
+          name: "Multiple Choice (master)",
+          native_height: 435,
+          native_width: 576,
+          no_snapshots: false,
+          show_delete_data_button: false,
+          thumbnail_url: "",
+          customizable: true,
+          authorable: true
+        }
+      },
+      type: "ManagedInteractive",
+      ref_id: "315-ManagedInteractive",
+      linked_interactives: [
+        {
+          ref_id: "314-ManagedInteractive",
+          label: "mc question"
+        }
+      ]
+    };
+
+    render(<ManagedInteractive
+              embeddable={sampleLinkedEmbeddable}
+              questionNumber={1}
+              setSupportedFeatures={mockSetSupportedFeatures}
+              setSendCustomMessage={mockSetSendCustomMessage}
+              setNavigation={mockSetNavigation}
+              />);
+    expect(screen.getByTestId("managed-interactive")).toBeInTheDocument();
+    // allow initialization to complete
+    jest.runAllTimers();
+
+    act(() => {
```

Comment:
I was just curious after @scytacki remarked about `act` in one of my PRs and it looks like it is not needed when we use (like we do in this repo) React Testing Library (@testing-library/react) as it is baked in to the helpers.

I don't think anything needs to be done for this PR but we should all be looking out for it in new tests.

More info: 

- https://reactjs.org/docs/testing-recipes.html#act
- https://javascript.plainenglish.io/you-probably-dont-need-act-in-your-react-tests-2a0bcd2ad65c

> You might find using act() directly a bit too verbose. To avoid some of the boilerplate, you could use a library like [React Testing Library](https://testing-library.com/react), whose helpers are wrapped with act().